Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -445,6 +445,13 @@ theorem NormedAddCommGroup.cauchy_series_of_le_geometric'' {C : ℝ} {u : ℕ 
     exact h _ H
 #align normed_add_comm_group.cauchy_series_of_le_geometric'' NormedAddCommGroup.cauchy_series_of_le_geometric''
 
+/-- The term norms of any convergent series are bounded by a constant. -/
+lemma norm_bounded_of_cauchy_series (h : CauchySeq fun n ↦ ∑ k in range n, f k) :
```

Comment:
I think this needs to be renamed a bit, `bounded` and `cauchy_series` aren't names so this seems very hard to discover using the naming convention at present

Reply:
I don't know; `cauchy_series` is in two other theorems in this file. `exists_norm_le_of_cauchySeq`?

Reply:
I think that those should also be changed, but let's not have that hold up this pr